### ISSUE TITLE
Extension breaks the system with typo3 exception

### ISSUE THREAD

**Issue by rickymathewkweb** [FriendsOfTYPO3/rsaauth#1]
On installing this extension on typo3 10.4 version it breaks with an 'undefined class constant ' error. See screenshot

![Screen Shot 2020-06-05 at 11 08 45](https://user-images.githubusercontent.com/59689479/83858806-52026400-a71d-11ea-83af-3a3a27926a05.png)

**Comment by lolli42** [comment]
has been fixed with https://github.com/FriendsOfTYPO3/rsaauth/commit/fe66706ac544118911c6522271bf2277a649bafa

**Comment by froemken** [comment]
yes, it's solved with current master, but a new release would be nice
